internal/bminventory/inventory.go

```diff
@@ -884,7 +885,7 @@ func (b *bareMetalInventory) GenerateClusterISOInternal(ctx context.Context, par
 	updates["image_expires_at"] = strfmt.DateTime(now.Add(b.Config.ImageExpirationTime))
 	updates["image_download_url"] = ""
 	updates["image_static_network_config"] = staticNetworkConfig
-	updates["image_mirror_registries_config"] = mirrorRegistiresConfig
+	updates["image_mirror_registries_config"] = mirrorRegistriesConfig
```

Comment:
Do we set this field unconditionally even if there are no mirrors?

Reply:
yes. The mirrorRegistriesConfig is createdusing the input from the image-create-params. in case no mirrors are provided, the string is empty